domains/certificates/Query.API/API/MeasurementsSyncer/MeteringPointSyncInfo.cs

```diff
@@ -1,5 +1,12 @@
 using System;
+using DataContext.ValueObjects;
 
 namespace API.MeasurementsSyncer;
 
-public record MeteringPointSyncInfo(string GSRN, DateTimeOffset StartSyncDate, string MeteringPointOwner);
+public record MeteringPointSyncInfo(string GSRN,
```

Comment:
You could wrap all these arguments into the Contract model

Reply:
Not sure what you mean

Reply:
It looks like all these parameters come from the "oldestContract" in contractState.cs 
So you could just make the method like this `MeteringPintSyncInfo(Contract contract)`

Reply:
That is too much info coming into syncInfo. Contract holds more info than needed

Reply:
Maybe i am missing some use cases, but IssuingContracts contains 
            Id = contract.Id,
            GSRN = contract.GSRN,
            StartDate = contract.StartDate.ToUnixTimeSeconds(),
            EndDate = contract.EndDate?.ToUnixTimeSeconds(),
            Created = contract.Created.ToUnixTimeSeconds(),
            MeteringPointType = contract.MeteringPointType,
            Technology = contract.Technology
            
           Why do we have this meteringpointsyncinfo model, it looks like a copy of issuingcontract?

Reply:
It's there for two different purposes. If contracts gets more logic we would have to expand sync infos as well. Besides sync info dont need created and endDate